lib/roo/excelx/sheet.rb

```diff
@@ -23,7 +23,16 @@ def cells
       end
 
       def present_cells
-        @present_cells ||= cells.select { |_, cell| cell && !cell.empty? }
+        @present_cells ||= begin
+          warn %{
+[DEPRECATION] present_cells is deprecated. Alternate:
```

Comment:
Spacing issues here?

Reply:
On purpose. Adding Indentation space to text wrapped in multiline %{} will also add space in o/p of depreciation warning. Possible alternate :
```
warn ".... \n"\
     ".... \n"\
     ".... "
```

I avoided this alternate because it is hard to read and you have to specify \n manually

Reply:
👍 
sounds good!